bpf/Makefile

```diff
@@ -267,7 +267,7 @@ tests/bpf_ct_tests.o: tests/bpf_ct_tests.ll
 
 .PHONY: go_prog_test
 go_prog_test: $(BPF_TEST)
-	sudo go test -tags privileged_tests ./tests/prog_test
+	sudo -E PATH="$(PATH)" $(GO) test -tags privileged_tests ./tests/prog_test
```

Comment:
nit: this might cause some files (e.g. go/pkg/mod//etc) having root permission, maybe one alternative is like below

```
go test -tags privileged_tests -exec sudo -v ./tests/prog_test
```

Reply:
Oh, nice :tada: TIL about `go test -exec`.